tedge/src/cli/connect/command.rs

```diff
@@ -251,6 +253,7 @@ async fn check_connection_azure() -> Result<(), ConnectError> {
 
 fn new_bridge(
     bridge_config: &BridgeConfig,
+    cloud: &Cloud,
```

Comment:
you name it here `cloud` and below it's `cloud_name`. Is there a difference? Also, why do you take a reference?

I would suggest to rename `cloud_name` below to just `cloud`